src/native.rs

```diff
@@ -1659,6 +1659,41 @@ impl HasContext for Context {
         );
     }
 
+    unsafe fn compressed_tex_sub_image_2d(
+        &self,
+        target: u32,
+        level: i32,
+        x_offset: i32,
+        y_offset: i32,
+        width: i32,
+        height: i32,
+        format: u32,
+        pixels: CompressedPixelUnpackData,
+    ) {
+        let gl = &self.raw;
+        gl.CompressedTexSubImage2D(
+            target,
+            level,
+            x_offset,
+            y_offset,
+            width,
+            height,
+            format,
+            match pixels {
```

Comment:
can we move this match out to get both values at once?

Reply:
Done!